src/SDKs/Azure.Core/data-plane/Azure.Core/Azure.Core.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is the implementation of the Azure Client Pipeline</Description>
+    <AssemblyTitle>Microsoft Azure Client Pipeline</AssemblyTitle>
+    <Version>1.0.0-preview.1</Version>
+    <PackageTags>Microsoft Azure Client Pipeline</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
```

Comment:
What should be here?

Reply:
Release notes are supposed to be her.

Reply:
Yes -- I left them empty for now -- we should put something there before we ship.

Reply:
I will file an issue for generating the release notes.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/5220